### ISSUE TITLE
Codacy

### ISSUE THREAD

**Issue by rugk** [PrivateBin/PrivateBin#592]
[This](https://github.com/PrivateBin/PrivateBin/pull/591#issuecomment-594533076) is annoying. It should use the proper GitHub APIs or whatnot to comment that, not _a personal user account_.

Can you, @elrido, maybe revoke your permission that this thing uses that? It is annoying.

**Comment by elrido** [comment]
I can and you can too - you had me install codacy in the first place. Should we remove it again, then? We need to also remove it from the BADGES.md

**Comment by rugk** [comment]
I just guess that they authenticate as _you_ (with a personal access token?) is a deprecated/legacy way of doing it, and I guess that needs to be setup in a better way. E.g. could you instead use the proper GItHub feature here: https://github.com/marketplace/codacy

**Comment by elrido** [comment]
Ok, I'll have a quick look at it on the weekend.

**Comment by elrido** [comment]
I've investigated this and all of the integrations got done via my account, not just codacy. In the case of codacy I could add the org account, but the integration would still go through my user account for the link. I did find an option to disable the commenting, though, which I did. So we will still get the status emails and status display in PRs, but the comments in my name should be silenced.

Thi link you provided is exactly what we have installed, BTW.
